lib/src/linux/qarma_and_zenity_handler.dart

```diff
@@ -57,8 +57,6 @@ class QarmaAndZenityHandler implements DialogHandler {
         return "Media Files | ${DialogHandler.toCaseInsensitive('*.avi *.flv *.mkv *.mov *.mp4 *.mpeg *.webm *.wmv *.bmp *.gif *.jpeg *.jpg *.png')}";
       case FileType.video:
         return "Video Files | ${DialogHandler.toCaseInsensitive('*.avi *.flv *.mkv *.mov *.mp4 *.mpeg *.webm *.wmv')}";
-      default:
```

Comment:
This is reported by https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=unreachable_switch_default#unreachable_switch_default